app/src/main/java/com/gacode/relaunchx/ReLaunch.java

```diff
@@ -1683,53 +1642,24 @@ public boolean onSingleTapConfirmed(MotionEvent e) {
 							200, 100, 0);
 					gv.dispatchTouchEvent(ev);
 				} else { // other devices
-					int first = gv.getFirstVisiblePosition();
-					int total = itemsArray.size();
-					int last = gv.getLastVisiblePosition();
-					if (total == last + 1)
-						return true;
-					int target = last + 1;
-					if (target > (total - 1))
-						target = total - 1;
-					RepeatedDownScroll ds = new RepeatedDownScroll();
-					ds.doIt(first, target, 0);
+					if (!prefs.getBoolean("doNotHyph", true)) {
+						setItemsHeights(gv);
+					}
+					gv.setSelection(gv.getLastVisiblePosition());
 				}
 				return true;
 			}
 
 			@Override
 			public boolean onDoubleTap(MotionEvent e) {
-				if (prefs.getBoolean("disableScrollJump", true) == false) {
-					int first = gv.getFirstVisiblePosition();
-					int total = itemsArray.size();
-					int last = gv.getLastVisiblePosition();
-					if (total == last + 1)
```

Comment:
Why do you want to remove this functionality?

Reply:
Becouse when do you want to scroll up/down two pages in a row (in short time) it can be interpreted as double tap.